src/receipt/receipt_std.rs

```diff
@@ -72,12 +72,19 @@ impl io::Read for Receipt {
                 bytes::store_number_unchecked(buf, *is);
             }
 
-            Self::Panic { id, reason, pc, is } => {
+            Self::Panic {
+                id,
+                reason,
+                pc,
+                is,
+                contract_id,
+            } => {
                 let buf = bytes::store_number_unchecked(buf, ReceiptRepr::Panic as Word);
 
                 let buf = bytes::store_array_unchecked(buf, id);
                 let buf = bytes::store_number_unchecked(buf, *reason);
                 let buf = bytes::store_number_unchecked(buf, *pc);
+                let buf = bytes::store_array_unchecked(buf, contract_id);
```

Comment:
The issue says the next:

> As noted in the specs, this field should not be included in canonical serialization. It's only an application-level concern and shouldn't be validated as part of the blockchain state. (ie. exclude from derive(eq) etc)

You don't need to do sterilization and deserialization. So better to make the field `Option<ContractId>` and fill it with `None`.

Reply:
Jup, ypu are right. At the beginning, I did it like that, but I couldn't serialize the Optional, so I left it like this, hoping that someone would give me an idea :D So Thanx!!